### PR TITLE
🌐 [i18n-KO] Translated `blip-2.md` to Korean

### PR DESCRIPTION
**Pull request by cjfghk5697**
# What does this PR do?

Translated the `blip-2.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

@jungnerd, @cjfghk5697, @yijun-lee, @mreraser

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@stevhliu May you please review this PR?

### CODE REVIEW

File: docs/source/ko/model_doc/blip-2.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+
+-->
+
+# BLIP-2
```
Comment (mreraser) [inline docs/source/ko/model_doc/blip-2.md:17]:
```suggestion
# BLIP-2[[blip-2]]
```

TOC가 누락되어 추가하였습니다.

File: docs/source/ko/model_doc/blip-2.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+## 개요[[overview]]
+BLIP-2 모델은 Junnan Li, Dongxu Li, Silvio Savarese, Steven Hoi의 [BLIP-2: Bootstrapping Language-Image Pre-training with Frozen Image Encoders and Large Language Models](https://arxiv.org/abs/2301.12597) 논문에서 제안되었습니다. BLIP-2는 동결된 사전 학습된 이미지 인코더와 대규모 언어 모델(LLM)을 활용하여, 그 사이에 12층 경량화한 Transformer 인코더를 학습시킴으로써 여러 비전-언어 작업에서 SOTA를 달성했습니다. 특히, BLIP-2는 800억 파라미터 모델인 Flamingo를 제로샷 VQAv2에서 8.7% 개선했으며, 학습 가능한 파라미터 수는 54배 적습니다.
```
Comment (mreraser) [inline docs/source/ko/model_doc/blip-2.md:20]:
```suggestion
BLIP-2 모델은 Junnan Li, Dongxu Li, Silvio Savarese, Steven Hoi의 [BLIP-2: Bootstrapping Language-Image Pre-training with Frozen Image Encoders and Large Language Models](https://arxiv.org/abs/2301.12597) 논문에서 제안되었습니다. BLIP-2는 동결된 사전 학습 이미지 인코더와 대규모 언어 모델(LLM)을 연결하는 12층의 경량 Transformer 인코더를 학습시켜, 여러 비전-언어 작업에서 SOTA(현재 최고의 성능)을 달성했습니다. 특히, BLIP-2는 800억 개의 파라미터를 가진 Flamingo 모델보다 제로샷 VQAv2에서 8.7% 더 높은 성능을 기록했으며, 학습 가능한 파라미터 수는 Flamingo보다 54배 적습니다.
```
조금 더 자연스러운 한국말로 바꿔보았습니다.

File: docs/source/ko/model_doc/blip-2.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+논문의 초록은 다음과 같습니다:
+
+*비전-언어 사전 학습의 비용은 대규모 모델의 엔드-투-엔드 학습으로 인해 점점 더 부담스러워지고 있습니다. 본 논문은 사전 학습된 이미지 인코더와 대규모 언어 모델을 활용하여 비전-언어 사전 학습을 부트스트래핑하는 일반적이고 효율적인 사전 학습 전략인 BLIP-2를 제안합니다. BLIP-2는 경량화된 Querying Transformer를 통해 모달리티 간의 차이를 연결하며, 두 단계로 사전 학습됩니다. 첫 번째 단계는 동결된 이미지 인코더로부터 비전-언어 표현 학습을 부트스트래핑하고, 두 번째 단계는 동결된 언어 모델로부터 비전-언어 생성 학습을 부트스트래핑합니다. BLIP-2는 기존 방법들에 비해 훨씬 적은 학습 가능한 파라미터로 다양한 비전-언어 작업에서 최첨단 성능을 달성합니다. 예를 들어, 우리 모델은 제로샷 VQAv2에서 Flamingo80B보다 8.7% 높은 성능을 기록하며, 학습 가능한 파라미터 수는 54배 적습니다. 우리는 또한 자연어 명령을 따를 수 있는 제로샷 이미지-텍스트 생성의 새로운 기능을 입증했습니다.*
+
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/model_doc/blip2_architecture.jpg"
+alt="drawing" width="600"/> 
+
+<small> BLIP-2 구조. <a href="https://arxiv.org/abs/2301.12597">논문 원본</a> 에서 발췌. </small>
```
Comment (mreraser) [inline docs/source/ko/model_doc/blip-2.md:29]:
```suggestion
<small> BLIP-2 구조. <a href="https://arxiv.org/abs/2301.12597">원본 논문</a> 에서 발췌. </small>
```
엄청 중요한건 아닌데, original paper는 `원본 논문`이 좀 더 맞지 않나 싶어 제안드립니다.

File: docs/source/ko/model_doc/blip-2.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/transformers/model_doc/blip2_architecture.jpg"
+alt="drawing" width="600"/> 
+
+<small> BLIP-2 구조. <a href="https://arxiv.org/abs/2301.12597">원본 논문</a> 에서 발췌. </small>
+
+이 모델은 [nielsr](https://huggingface.co/nielsr)가 기여했습니다. 원본 코드는 [여기](https://github.com/salesforce/LAVIS/tree/5ee63d688ba4cebff63acee04adaef2dee9af207)에서 확인할 수 있습니다.
+
+## 사용 팁[[usage-tips]]
+
+- BLIP-2는 이미지와 조건에 따라 텍스트 프롬프트를 입력받아 조건부 텍스트 생성합니다. 추론 시 [`generate`] 메소드를 사용하는 것이 권장됩니다.
```
Comment (yijun-lee) [inline docs/source/ko/model_doc/blip-2.md:35]:
```suggestion
- BLIP-2는 이미지와 조건에 따라 텍스트 프롬프트를 입력받아 조건부 텍스트를 생성합니다. 추론 시 [`generate`] 메소드를 사용하는 것이 권장됩니다.
```

Comment (yijun-lee) [reply docs/source/ko/model_doc/blip-2.md:35]: '조건부 텍스트를 생성합니다' 혹은 '조건부 텍스트 생성을 수행합니다'가 보다 자연스러울 것 같습니다!

Comment (yijun-lee) [reply docs/source/ko/model_doc/blip-2.md:35]: 이외에는 완벽한 것 같습니다. 수고 많으셨습니다 :)

**Review by yijun-lee — CHANGES_REQUESTED** [review]

**Review by stevhliu — APPROVED** [review]
Thanks, we can merge once the conflict is resolved! 🤗

**Pull request merged**